Lib/test/test_unicodedata.py

```diff
@@ -289,8 +289,7 @@ def test_failed_import_during_compiling(self):
         # We use a separate process because the unicodedata module may already
         # have been loaded in this process.
         result = script_helper.assert_python_failure("-c", code)
-        error = "SyntaxError: (unicode error) \\N escapes not supported " \
-            "(can't load unicodedata module)"
+        error = 'ImportError: PyCapsule_Import could not import module "unicodedata"'
```

Comment:
I think we want to keep the previous error message. The message about `\N` escapes makes sense in `_PyUnicode_DecodeUnicodeEscapeInternal` in `Objects/unicodeobject`.

Reply:
Sounds reasonable.
Done.